vite.main.config.ts

```diff
@@ -7,12 +7,33 @@ import {
   pluginHotRestart,
 } from './vite.base.config'
 import { sentryVitePlugin } from '@sentry/vite-plugin'
+import * as dotenv from 'dotenv'
+
+function getDotEnv(defaults: Record<string, string>) {
+  const env = {
+    ...defaults,
+    ...dotenv.config().parsed,
+  }
+
+  return Object.fromEntries(
+    Object.entries(env).map(([key, value]) => [key, JSON.stringify(value)])
+  )
+}
 
 // https://vitejs.dev/config
 export default defineConfig((env) => {
   const forgeEnv = env as ConfigEnv<'build'>
   const { forgeConfigSelf } = forgeEnv
-  const define = getBuildDefine(forgeEnv)
+
+  const define = {
+    ...getBuildDefine(forgeEnv),
+    ...getDotEnv({
+      GRAFANA_CLIENT_ID: '<tbd>',
```

Comment:
We don't have a client id in prod. Not sure if we should commit it either (although the client id is not meant to be a secret).

Reply:
We could use github variables to insert it so that it can be changed ahead of build time 🤔

Reply:
Yeah, but that's one more step of setup... 

Thinking on it, I think we should just commit it to code. It's not secret. Once the binary is built anybody with enough motivation and access to Google can read it. 

Jumping through hoops to hide it adds complexity and all we achieve is security by obscurity.